From bd8c2c9c3f01af262fcd8b94a539caf0283cca54 Mon Sep 17 00:00:00 2001
From: Ikey Doherty <michael.i.doherty@intel.com>
Date: Sat, 12 Mar 2016 10:43:37 +0000
Subject: [PATCH] Use rpcbind for sunrpc services

Signed-off-by: Ikey Doherty <michael.i.doherty@intel.com>
---
 etc-services | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/etc-services b/etc-services
index 704cd12..3579f0a 100644
--- a/etc-services
+++ b/etc-services
@@ -72,8 +72,8 @@ pop2		109/tcp		postoffice pop-2 # POP version 2
 pop2		109/udp		pop-2
 pop3		110/tcp		pop-3		# POP version 3
 pop3		110/udp		pop-3
-sunrpc		111/tcp		portmapper	# RPC 4.0 portmapper
-sunrpc		111/udp		portmapper
+sunrpc		111/tcp		rpcbind		# RPC 4.0 portmapper
+sunrpc		111/udp		rpcbind
 auth		113/tcp		authentication tap ident
 sftp		115/tcp
 uucp-path	117/tcp
-- 
2.7.1

